staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_history.go

```diff
@@ -177,6 +177,11 @@ func (o *RolloutHistoryOptions) Run() error {
 			}
 
 			if o.Revision > 0 {
+				// Ensure the specified revision exists before printing
+				if historyInfo[o.Revision] == nil {
```

Comment:
Idiomatic go way of checking for map key existence is the second param of key access...
https://stackoverflow.com/a/2050629

So something like this...
```
revision, exists := historyInfo[o.Revision]
if !exists {

}
```